api/lib/infrastructure/repositories/organization-participant-repository.js

```diff
@@ -6,6 +6,20 @@ const CampaignTypes = require('../../domain/models/CampaignTypes');
 const CampaignParticipationStatuses = require('../../domain/models/CampaignParticipationStatuses');
 
 async function getParticipantsByOrganizationId({ organizationId, page, filters = {} }) {
+  const { count } = await knex
+    .select(knex.raw('COUNT(DISTINCT "organization-learners"."id")'))
+    .from('organization-learners')
+    .join('campaign-participations', function () {
+      this.on('campaign-participations.organizationLearnerId', 'organization-learners.id').andOn(
+        'campaign-participations.deletedAt',
+        knex.raw('IS'),
+        knex.raw('NULL')
+      );
+    })
+    .where({ organizationId: organizationId, isDisabled: false })
+    .first();
+  const totalParticipants = count ? count : 0;
```

Comment:
On peut utiliser le [nullish coalescing operator](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)
```suggestion
  const totalParticipants = count ?? 0;
```

Reply:
c'est joli 🤩 
Merci !